src/openapi.yaml

```diff
@@ -1,7 +1,7 @@
 openapi: 3.0.3
 info:
   title: Open Forms API
-  version: 2.6.0-alpha.0
+  version: 2.6.0
```

Comment:
This doesn't have the `-beta.1`?

Reply:
Deliberate - API is no longer changing and this PR is for the 2.6.0 stable release :) The beta release are just git tags/image tags for testing purposes.